.releaserc

```diff
@@ -0,0 +1,26 @@
+{
+  "ci": false,
+  "branches": ["main", {"name": "alpha", "prerelease": true}, {"name": "beta", "prerelease": true}],
+  "plugins": [
+    ["@semantic-release/commit-analyzer", {
+      "preset": "angular",
+      "releaseRules": [
+        {"type": "dep", "release": "patch"}
+      ],
+      "parserOpts": {
+        "noteKeywords": ["BREAKING CHANGE", "BREAKING-CHANGE"]
```

Comment:
I think you mentioned in our internal ticket to use `breaking-change:` as well as trailer, or is that case insensitive?

Reply:
after some discussion we decided to use the upper-case version to put emphasize on the importance of the breaking change. Also the [spec](https://www.conventionalcommits.org/en/v1.0.0/#specification) explicitly states that 
>The units of information that make up Conventional Commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.